### DISCUSSION TITLE
We found an error during import

### DISCUSSION THREAD

**Discussion by youcefdilmi88** [community/community#39786] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How to fix this please:
![77BB0D08-4DCA-4607-9150-64B09BA122D2](https://user-images.githubusercontent.com/57549784/203408074-8d224772-d9b6-40ff-9e8a-9e329d748851.png)

**Comment by ossanna16** [comment]
Hi there @youcefdilmi88 and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
